docs/reference/settings/dlm-settings.asciidoc

```diff
@@ -0,0 +1,18 @@
+[role="xpack"]
+[[dlm-settings]]
+=== {dlm-cap} settings in {es}
+[subs="attributes"]
+++++
+<titleabbrev>{dlm-cap} settings</titleabbrev>
+++++
+
+preview::[]
+
+These are the settings available for configuring <<data-lifecycle-management, {dlm}>> ({dlm-init}).
+
+==== Cluster level settings
+
+[[indices-dlm-poll-interval]]
```

Comment:
Would we want to document the automatic rollover setting here? I think on prem clusters would benefit from it?

Reply:
ah yes, I missed it.